applications/app/services/VideoSiteMap.scala

```diff
@@ -53,6 +53,8 @@ class VideoSiteMap(contentApiClient: ContentApiClient) {
 
   def getLatestContent()(implicit executionContext: ExecutionContext): Future[NodeSeq] = {
 
+    val date = DateTime.now(DateTimeZone.UTC).minusDays(2)
```

Comment:
why not calling it inline? it is used only once

Reply:
Just to make it more readable